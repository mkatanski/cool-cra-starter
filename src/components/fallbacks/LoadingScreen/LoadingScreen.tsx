import React from "react";

import styled from "styled-components";

const Container = styled.div`
  align-items: center;
  display: flex;
  flex-direction: column;
  height: 100%;
  justify-content: center;
  min-height: 100%;
  padding: 10px;
`;

export const LoadingScreen: React.FC = () => <Container>Loading...</Container>;
