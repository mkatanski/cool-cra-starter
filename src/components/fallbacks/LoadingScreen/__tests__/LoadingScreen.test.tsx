import React from "react";
import TestRenderer from "react-test-renderer";
import "jest-styled-components";

import { LoadingScreen } from "../LoadingScreen";

describe("components/fallbacks", () => {
  describe("LoadingScreen", () => {
    it("should render LoadingScreen component", () => {
      const tree = TestRenderer.create(<LoadingScreen />);

      expect(tree).toMatchSnapshot();
    });
  });
});
