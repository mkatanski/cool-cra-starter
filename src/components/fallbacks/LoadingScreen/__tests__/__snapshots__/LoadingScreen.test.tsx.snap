// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/fallbacks LoadingScreen should render LoadingScreen component 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-height: 100%;
  padding: 10px;
}

<div
  className="c0"
>
  Loading...
</div>
`;
